pytorch_lightning/plugins/training_type/rpc.py

```diff
@@ -63,16 +63,15 @@ def init_rpc_connection(self, global_rank: int, world_size: int) -> None:
         rpc._set_rpc_timeout(self.rpc_timeout_sec)
         self._is_rpc_initialized = True
 
-    def rpc_save_model(self, save_model_fn, last_filepath, trainer, pl_module) -> None:
+    def rpc_save_model(self, trainer, save_model_fn: Callable, filepath: str) -> None:
```

Comment:
re-ordering the function args here breaks backwards compatibility

Reply:
I needed to remove `pl_module` so also chose to reorder for consistency with other hooks.

@justusschock (author of this line) should I add BC support + deprecation warning for this?

Reply:
RPC is really experimental, and only DDPSequential is using this API. I think it is safe to assume it won't break people code.